src/main/java/org/elasticsearch/search/aggregations/bucket/histogram/DateHistogramParser.java

```diff
@@ -105,10 +104,8 @@ public AggregatorFactory parse(String aggregationName, XContentParser parser, Se
                     preZone = DateTimeZone.forID(parser.text());
                 } else if ("post_zone".equals(currentFieldName) || "postZone".equals(currentFieldName)) {
                     postZone = DateTimeZone.forID(parser.text());
-                } else if ("pre_offset".equals(currentFieldName) || "preOffset".equals(currentFieldName)) {
-                    preOffset = parseOffset(parser.text());
-                } else if ("post_offset".equals(currentFieldName) || "postOffset".equals(currentFieldName)) {
-                    postOffset = parseOffset(parser.text());
+                } else if ("offset".equals(currentFieldName)) {
```

Comment:
Option 2 is the plan, see discussion on https://github.com/elasticsearch/elasticsearch/issues/9062